src/gettext_plural_form.cpp

```diff
@@ -99,7 +99,7 @@ static ParserResult reduce_ltr(const size_t nplurals, const ParserResult &res, c
 {
```

Comment:
unrelated but it appears whoever reviewed the code in this file did not notice the `const wstring_view&` anti-pattern

Reply:
I might as well fix it in this PR.